packages/workflow/src/interfaces.ts

```diff
@@ -81,11 +81,15 @@ export interface WorkflowInfo {
    */
   continuedFromExecutionRunId?: string;
 
-  // TODO expose from Core
   /**
-   * Time at which the Workflow Run started
+   * Time at which this Workflow Execution Chain was started
```

Comment:
Do we have a stable link from our docs to the concept of run vs chain?

Reply:
```suggestion
   * Time at which this [Workflow Execution Chain](https://docs.temporal.io/workflows#workflow-execution-chain) was started
```